grpc/src/main/java/com/linecorp/armeria/server/grpc/ArmeriaServerCall.java

```diff
@@ -277,34 +277,35 @@ public void setCompression(String compressorName) {
 
     @Override
     public void messageRead(ByteBufOrStream message) {
-        boolean success = false;
+
         final I request;
 
+        boolean success = true;
```

Comment:
I guess you can still start with `success = false`, since the ownership transfer after this try block?

Reply:
Updated